Cargo.toml

```diff
@@ -6,11 +6,15 @@ description = "Pseudo Terminal (PTY) plugin for Tauri"
 keywords = ["tauri-plugin"]
 license = "MIT"
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.77.2"
 exclude = ["/examples"]
+links = "tauri-plugin-pty"
 
 [dependencies]
-portable-pty = { version = "0.8.1", features = ["serde_support"] }
-serde = "1.0.193"
-serde_json = "1.0.108"
-tauri = { version = "^1.5.4" }
+portable-pty = { version = "0.9.0", features = ["serde_support"] }
+serde = "1.0"
+serde_json = "1.0"
+tauri = { version = "^2" }
+
+[build-dependencies]
+tauri-plugin = { version = "2.0.3", features = ["build"] }
```

Comment:
Maybe this version should be "2" for consistency

Reply:
thanks, fixed!